lisa/sut_orchestrator/azure/platform_.py

```diff
@@ -224,6 +232,30 @@ class AzureLocation:
     capabilities: Dict[str, AzureCapability] = field(default_factory=dict)
 
 
+@dataclass_json()
+@dataclass
+class CloudEndpointsSchema:
```

Comment:
can it directly inherits from `msrestazure.azure_cloud`?

Reply:
I tried like below
`class CloudEndpointsSchema(CloudEndpoints):`
And see below pylance error
`Base class type is unknown, obscuring type of derived class`
I don't understand the benefit of inherit.

Reply:
ok, if it doesn't support, we can keep as is. Use the same object can reduce maintenance.